package.json

```diff
@@ -45,10 +45,10 @@
     "@empathyco/x-deep-merge": "^1.3.0-alpha.1",
     "@empathyco/x-translations": "^1.1.0-alpha.4",
     "@rollup/plugin-buble": "~0.21.3",
-    "@rollup/plugin-commonjs": "~14.0.0",
+    "@rollup/plugin-commonjs": "~21.0.1",
     "@rollup/plugin-json": "~4.1.0",
-    "@rollup/plugin-node-resolve": "~8.4.0",
-    "@rollup/plugin-replace": "~2.3.2",
+    "@rollup/plugin-node-resolve": "~13.1.3",
+    "@rollup/plugin-replace": "~3.0.1",
     "@types/autoprefixer": "~9.6.1",
```

Comment:
We cannot update cssnano to the latest version. If we update to this version, we need to update the vue plugin to the latest version because in the current version they are using `"postcss": "^7.0.11"` and If we update css and autoprefixer to the latest version, we need to update postcss to version 8, but vue plugin is not working with it.